.github/workflows/test.yml

```diff
@@ -89,15 +89,18 @@ jobs:
           # test default settings with 3.11 across all supported evm versions
           - evm-version: london
           - evm-version: paris
-          - evm-version: cancun
+          - evm-version: shanghai
 
-          # py-evm rules
+          # test py-evm
           - evm-backend: py-evm
+            evm-version: shanghai
+
           - evm-backend: py-evm
             evm-version: cancun
```

Comment:
```suggestion
```

Reply:
i'd rather have the clarity, since evm-version is being changed on this axis